tests/e2e/instrumentation-apache-httpd/00-install-instrumentation.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: opentelemetry.io/v1alpha1
+kind: Instrumentation
+metadata:
+  name: apache
+spec:
+  exporter:
+    endpoint: http://localhost:4317
```

Comment:
given we default here, can we remove this from the install which will help confirm that expectation?